### PR TITLE
+ better select questions

### LINKED ISSUE #73: Update how selecting questions works

### PR DESCRIPTION
**Pull request by weining-li**
Resolves #73

### CODE REVIEW

File: frontend/src/components/home-page/content-tree.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .content-tree {
   width: 40%;
+  max-width: 400px;
```
Comment (Goff-Davis) [inline frontend/src/components/home-page/content-tree.scss:3]: This should be a bit wider probably. It makes the questions pretty squished.

File: frontend/src/components/home-page/home.tsx [hunk 2]
```diff
@@ -1,18 +1,70 @@
...
+  const dispatch = global.dispatch;
+  const state = global.state;
+
+  const toggleEditQuestion = () => {
+    dispatch({ type: "edit-preview-question" });
+    dispatch({ type: "open-creator" });
+  };
+
+  const deleteQuestion = () => {
+    // const newQuestions = state.questions.slice();
```
Comment (Goff-Davis) [inline frontend/src/components/home-page/home.tsx:21]: Is this commented code worth keeping?

File: frontend/src/components/home-page/question-types/multiple-choice.tsx [hunk 3]
```diff
@@ -29,27 +29,33 @@ const AnswerChoice = ({
   return (
     <div
       className={`answer-choice ${
-        showCorrectPreviewResponse ? (correct ? "correct" : "incorrect") : ""
```
Comment (Goff-Davis) [inline frontend/src/components/home-page/question-types/multiple-choice.tsx:32]: Can you split this file into two separate files? One for the `MultipleChoice` component and one for the `AnswerChoice` component? We shouldn't combine components in the same file, even if they are used once.

File: frontend/src/components/home-page/content-tree.scss [hunk 4]
```diff
@@ -1,5 +1,6 @@
 .content-tree {
```
Comment (Goff-Davis) [inline frontend/src/components/home-page/content-tree.scss:1]: The styling for the home page needs to be fixed, particularly for wider screens. I don't think it's worth blocking the merge though for that.

**Review by Goff-Davis — CHANGES_REQUESTED** [review]

File: frontend/src/components/home-page/content-tree.scss (hunk 1 above)
Comment (weining-li) [reply frontend/src/components/home-page/content-tree.scss:3]: i thought i removed this smh

File: frontend/src/components/home-page/question-types/multiple-choice.tsx (hunk 3 above)
Comment (weining-li) [reply frontend/src/components/home-page/question-types/multiple-choice.tsx:32]: i wanna combine them into one component tbh :cake:

Comment (Goff-Davis) [reply frontend/src/components/home-page/question-types/multiple-choice.tsx:32]: That works too

File: frontend/src/components/home-page/content-tree.scss (hunk 4 above)
Comment (weining-li) [reply frontend/src/components/home-page/content-tree.scss:1]: this is discrimination cuz my screen isn't wide :<

**Review by Goff-Davis — CHANGES_REQUESTED** [review]
I found a bug with previewing questions. If you click on the second question down, it says Where is the captial of florida instead of what the actual question is. It doesn't happen with other questions though.

Otherwise LGTM. There is still the issue of styles with larger screens, particularly button and font sizes, I think because it goes off of vw or something. We can worry about that after we get design feedback though.

**Pull request merged**